ci/fpm-deployment.sh

```diff
@@ -34,6 +35,10 @@ prune=(
   "$destdir/src/f18estop.f90"
 )
 
+if [ ${maxrank} -lt 8 ]; then
+  prune+=("$destdir/test/test_mean_f03.f90")
+fi
+
```

Comment:
@jalvesz with #758 these lines should not be needed anymore. Could you check it, please (and remove them if needed)?

Reply:
Great, it worked locally, I just pushed the rollback on these lines